### PR TITLE
feat: add javascript language and frameworks

### PR DESCRIPTION
**Pull request by jalalazimi**

### CODE REVIEW

File: technologies/nodejs.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: "Nodejs"
+ring: "adopt"
+quadrant: "framework"
+isNew: "false"
+---
+
+Node.js is a no- browser JavaScript execution runtime. Its basis is Google's V8 engine. Node is event-driven and follows a non-blocking I/O model. It’s a good choice for restful APIs, realtime purposes or situations where many concurrent connections are expected, where each connection has a lightweight memory footprint.
+Node allows separation of concerns by using its package manager npm, which is also the largest ecosystem of Open Source libraries (modules). Modules are added as dependencies and offer a wide range of functionalities in a range from simple helper functions to mature web frameworks such as express.js.
+Many PaaS providers (AWS, Google Cloud Platform, Azure) support node, including deployment and monitoring services out of the box for scalable stateless applications.
```
Comment (ilvar) [inline technologies/nodejs.md]: Are we going to use Node? Where?

File: technologies/pose.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+name: "Pose"
+ring: "adopt"
+quadrant: "frameworks"
```
Comment (ilvar) [inline technologies/pose.md]: Not sure if it can be classified as a framework

Comment (jalalazimi) [reply technologies/pose.md]: according to the framework definition on Readme (`Useful libraries used in services to reduce boilerplate code, etc.`), it's a framework.

File: technologies/nodejs.md (hunk 1 above)
Comment (jalalazimi) [reply technologies/nodejs.md]: so now we use nodejs in Clustering API and on all of the UI projects for building pieces of stuff.

File: technologies/yarn.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+name: "Yarn"
+ring: "adopt"
+quadrant: "tools"
+isNew: "false"
+---
+
+Yarn was built to solve some known problems of npm. Since the release of yarn, npm has also solved a lot of those issues, but yarn is still faster, has a more consistent interface and is more reliable.
```
Comment (mdibaiee) [inline technologies/yarn.md]: I've had my own share of issues with Yarn's lockfile handling.
https://engineering.mixmax.com/blog/to-yarn-and-back-again-npm

I remember having this issue a few weeks ago with `news_api_admin_panel` and so I strongly suggest npm over yarn.

**Pull request merged**